actions/produce.py

```diff
@@ -36,9 +37,36 @@ def run(self, topic, message, hosts=None):
         # set default for empty value
         _client_id = self.config.get('client_id') or self.DEFAULT_CLIENT_ID
 
-        producer = KafkaProducer(bootstrap_servers=_hosts, client_id=_client_id)
+        if self.config.get('tls_enable', False):
+            producer = KafkaProducer(bootstrap_servers=_hosts, client_id=_client_id,
+                                     security_protocol='SSL',
+                                     ssl_cafile=self.file_parameter(self.config.get('tls_ca_certificate', None), '/var/tls_ca_certificate'),
+                                     ssl_certfile=self.file_parameter(self.config.get('tls_client_certificate', None), '/var/tls_client_certificate'),
+                                     ssl_keyfile=self.file_parameter(self.config.get('tls_client_key', None),'/var/tls_client_key'))
+        else:
+            producer = KafkaProducer(bootstrap_servers=_hosts, client_id=_client_id)
+
         producer.send(topic, message.encode('utf_8'))
         producer.flush()
         producer.close()
         return 0
+    
+    def file_parameter(self, contents, file_name):
```

Comment:
Can we rename the method (`create_file`, `create_file_base64`, etc)?

Reply:
OK